### PR TITLE
affine-cipher: add new problem

### PR DESCRIPTION
**Pull request by guygastineau**
I think I have everything together.  ran local-status; configlet uuid; configlet lint; updated json.
- I haven't figured out how to make test generators yet so I made this one by hand based off of the .json in problem-specifications.
- hopefully I can learn how to build the generators in the future

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -525,6 +525,14 @@
...
       "topics": null
     },
+    {
+      "slug": "affine-cipher",
+      "uuid": "d1267415-aff5-411d-b267-49a4a2c8fda2",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": null
+    },
```
Comment (Insti) [inline config.json]: Please add new exercises to the end of the config file.

Make sure you list the topics this exercise covers.

**Comment by Insti** [comment]
Cool. Thanks!

**Comment by Insti** [comment]
Have you read: https://github.com/exercism/ruby#implementing-a-generator

File: exercises/affine-cipher/.meta/hints.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Advanced
```
Comment (Insti) [inline exercises/affine-cipher/.meta/hints.md]: Is `hints.md` still used?

**Comment by Insti** [comment]
Problem specifications link: https://github.com/exercism/problem-specifications/tree/master/exercises/affine-cipher

**Comment by Insti** [comment]
OK, we can add the exercise without a generator for now and someone can add one later.

A lot of the existing exercises don't have topics because topics are a (relatively) new feature, but we like to make sure that any new exercises have all the up to date information. 

I think `hints.md` is another leftover from the v1 site, and now in v2 everything should just be included directly in the `README.md`

**Comment by guygastineau** [comment]
@insti thank you again for this help.
- I am back at my computer so I'll go ahead and make changes.
- Is it okay to add the hints.md info into the README manually?  I didn't put that info into the PR for `problem-specifications`, because not all languages can implement the hard way first without considerable difficulty.

- Also, I started reading about ERB so I think I will be able to make a generator within a week or two :smile:

**Comment by Insti** [comment]
> I started reading about ERB so I think I will be able to make a generator within a week or two 

You don't really need to know much about ERB have a look at some of the existing generators, they're mostly pretty standard.

**Comment by Insti** [comment]
Merged. 
Now people can try it out, and hopefully find their way here if they need to report any problems.

You can make a new PR when you feel up to trying to make a generator.

Thanks @guygastineau :heart:

**Comment by guygastineau** [comment]
Thanks for giving me direction for good resources and next steps.

**Pull request merged**